public/assets/scss/_index.scss

```diff
@@ -281,3 +281,228 @@ h5 .btn-link {
 .break {
     display: inline-block;
 }
+
+/* big tablets */
+@media only screen and (max-width: 1024px) {
+    header,
+    .holding,
+    .about,
+    .phone-div,
+    .faq {
+        padding-left: 3rem;
+    }
+    .benefits {
+        padding: 3rem;
+        height: 40rem;
+    }
+    .slide {
+        width: 13.5rem;
+    }
+    .prev,
+    .next {
+        top: 25%;
+    }
+}
+
+/* medium sized devices ie. tablets */
+@media only screen and (max-width: 991.99px) {
```

Comment:
Any special reason for using 991.99 here. Can we stick with bootstrap default devices and only use custom sizes when it is absolutely needed. 
```
Small devices (landscape phones, 576px and up)
@media (min-width: 576px) { ... }

// Medium devices (tablets, 768px and up)
@media (min-width: 768px) { ... }

// Large devices (desktops, 992px and up)
@media (min-width: 992px) { ... }

// Extra large devices (large desktops, 1200px and up)
@media (min-width: 1200px) { ... }
```
https://getbootstrap.com/docs/4.0/layout/overview/